.github/workflows/build-and-test.yml

```diff
@@ -133,7 +133,7 @@ jobs:
             - name: Run model tests
               shell: bash -l {0}
               run: |
-                  pip install --isolated --upgrade --only-binary natcap.invest --find-links=dist natcap.invest
+                  pip install $(find dist -name "natcap.invest*.whl")
```

Comment:
I might not be remembering correctly, but didn't we already make this change in a prior PR? This update looks familiar to me is all.

Reply:
Yeah, I had remembered the same thing and was surprised that I needed to do it once again.  Looking back through the PR queue, looks like #222 did something related, but only for sdists, I guess, and even then only in `release/3.9`.  This PR uses the same thinking for wheels as well as as sdists.  Pip's approach _should_ work, but the `bash`/`find` approach is just so much more expressive AND it works as we expect!